packages/bot-web-ui/src/components/dashboard/dashboard-component/dashboard-component.tsx

```diff
@@ -33,30 +74,51 @@ const DashboardComponent = ({
                 })}
             >
                 <div className='tab__dashboard__content'>
-                    <div className='tab__dashboard__centered'>
-                        <div className='tab__dashboard__header'>
-                            <Text color='prominent' line_height='xxl' size={is_mobile ? 's' : 'm'} weight='bold'>
-                                {localize('Load or build your bot')}
-                            </Text>
-                        </div>
+                    <div
+                        className={classNames('tab__dashboard__centered', {
+                            'tab__dashboard__centered--listed': !is_mobile && has_dashboard_strategies,
+                            'tab__dashboard__centered--not-listed': !has_dashboard_strategies,
+                        })}
+                    >
+                        {!has_dashboard_strategies && !is_mobile && (
```

Comment:
Isn't better approach to use `<MobileWrapper>` inside components instead of using boolean? Meets many times below, increase cognitive complexity

Reply:
@NikitK-deriv In this case, we need to prescribe how the functionality will work in this or that condition. Reducing the cognitive load implies simplifying the component, and reusing parts, which is what is done in this component. If you look at what it does`<MobileWrapper>`, you will see that it only checks for mobile version or not and returns component if mobile version. I.e. the same as what I have written. I could use `<Desktop wrapper` here, but I preferred using the `is_mobile` variable since I'm using it for other checks anyway.